src/plugins/content_management/server/plugin.ts

```diff
@@ -21,22 +21,37 @@ import {
   ContentManagementServerStart,
   SetupDependencies,
 } from './types';
+import { EventStreamService, EsEventStreamClientFactory } from './event_stream';
 import { procedureNames } from '../common/rpc';
 
-type CreateRouterFn = CoreSetup['http']['createRouter'];
-
 export class ContentManagementPlugin
   implements Plugin<ContentManagementServerSetup, ContentManagementServerStart, SetupDependencies>
 {
   private readonly logger: Logger;
   private readonly core: Core;
+  readonly #eventStream: EventStreamService;
+
+  constructor(initializerContext: PluginInitializerContext) {
+    const kibanaVersion = initializerContext.env.packageInfo.version;
 
-  constructor(initializerContext: { logger: PluginInitializerContext['logger'] }) {
     this.logger = initializerContext.logger.get();
-    this.core = new Core({ logger: this.logger });
+    this.#eventStream = new EventStreamService({
+      logger: this.logger,
+      clientFactory: new EsEventStreamClientFactory({
+        baseName: '.kibana',
```

Comment:
This may no longer be a concern, but we've used some config value instead of hardcoding `.kibana` for multi-tenant Kibana support.  So separate tenants would be using their own indices ...

Reply:
I was using the `core.savedObjects.getKibanaIndex()` at first, as ti is used in Event Log, but then decided to simplify to `.kibana`.

My reasoning was that indices that start with `.kibana*` are hardcoded in Elasticsearch to be a special indices that only Kibana can access. So I wanted to make it explicit to start with `.kibana`.

I'll take a look at the multi-tenant support feature. If it still exists, then the Event Stream should support it. But depending on `core.savedObjects.getKibanaIndex()` does not sound right to me. It is up to the Saved Objects implementation where they store their stuff and the Event Stream should not depend on that.

Reply:
I've added a comment in the Event Stream describing that `.kibana*` prefix has a special treatment in Elasticsearch.